project/AkkaBuild.scala

```diff
@@ -125,6 +125,19 @@ object AkkaBuild extends Build {
     dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm", slf4j % "test->compile")
   ) configs (MultiJvm)
 
+  lazy val clusterTools = Project(
+    id = "akka-cluster-tools",
+    base = file("akka-cluster-tools"),
+    dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm")
+  ) configs (MultiJvm)
+  
+  lazy val clusterSharding = Project(
+    id = "akka-cluster-sharding",
+    base = file("akka-cluster-sharding"),
+    dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm", 
+        persistence % "compile;test->provided", clusterTools)
```

Comment:
My reading of such things is «the "test" config of this one depends on the "provided" configuration of that one», which seems consistent with the observations.